packaging/nnstreamer.spec

```diff
@@ -322,6 +322,13 @@ Requires:	capi-ml-common-devel
 %description -n capi-nnstreamer-devel
 Developmental kit for Tizen Native NNStreamer API.
 
+%package -n capi-nnstreamer-devel-static
+Summary:    Static library for Tizen Native API
+Group:		Multimedia/Framework
+Requires:	capi-nnstreamer-devel = %{version}-%{release}
+%description -n capi-nnstreamer-devel-static
+Static library for Tizen Native API
+
```

Comment:
need separation for nnstreamer-devel as well. (look at line 645-659)

Reply:
ACK!